instagram_login_screen/lib/screens/Signup/signup_screen.dart

```diff
@@ -8,63 +9,74 @@ class SignupScreen extends StatefulWidget {
+class _SignupScreenState extends State<SignupScreen> with Utils {
   @override
   Widget build(BuildContext context) {
-    return MaterialApp(
-      debugShowCheckedModeBanner: false,
-      home: Scaffold(
-        body: Container(
-          height: double.infinity,
-          width: double.infinity,
-          decoration: const BoxDecoration(
-            gradient: LinearGradient(
-              begin: Alignment.topCenter,
-              end: Alignment.bottomCenter,
-              colors: [
-                Colors.purple,
-                Colors.blue,
-              ],
-            ),
+    return Scaffold(
+      body: Container(
+        height: double.infinity,
+        width: double.infinity,
+        decoration: const BoxDecoration(
+          gradient: LinearGradient(
+            begin: Alignment.topCenter,
+            end: Alignment.bottomCenter,
+            colors: [
+              Colors.purple,
+              Colors.blue,
+            ],
           ),
-          child: Padding(
-            padding: const EdgeInsets.only(
-              top: 10,
-              bottom: 10,
-              left: 30,
-              right: 30,
-            ),
-            child: SingleChildScrollView(
-              reverse: true,
-              child: Column(
-                mainAxisAlignment: MainAxisAlignment.spaceBetween,
-                children: <Widget>[
-                  Column(
-                    children: [
-                      const Icon(
-                        Icons.add_a_photo_outlined,
-                        size: 120,
-                        color: Colors.white70,
-                      ),
-                      const SizedBox(height: 30),
-                      buildTextField(text: 'Email'),
-                      const SizedBox(height: 20),
-                      buildTextField(text: 'Password', hide: true),
-                      const SizedBox(height: 20),
-                      buildTextField(text: 'Confirm Password', hide: true),
-                      const SizedBox(height: 20),
-                      buildTextField(text: 'Fullname'),
-                      const SizedBox(height: 20),
-                      buildTextField(text: 'Username'),
-                      const SizedBox(height: 20),
-                      buildButton(text: 'Sign Up'),
-                      const SizedBox(height: 20),
-                    ],
+        ),
+        child: Padding(
+          padding: const EdgeInsets.symmetric(
+            horizontal: 30,
+            vertical: 20,
+          ),
+          child: Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: <Widget>[
+              const Icon(
+                Icons.add_circle_outlined,
+                size: 120,
+                color: Colors.white70,
+              ),
+              const SizedBox(height: 50),
+              buildTextField(text: 'Email'),
+              const SizedBox(height: 25),
```

Comment:
bọc buildTextField trong Padding

Reply:
This 'signup_screen.dart' has been deprecated and deleted.